### PR TITLE
add license tests to structure tests

### PR DESCRIPTION
**Pull request by sharifelgamal**
License tests are automated tests to check that all packages installed in the image conform to Google's open source licensing stardards. Check out https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/structure_tests/licenses_test.go (and the corresponding README) for more details.

### CODE REVIEW

**Comment by kjin** [comment]
Are these tests run when we update the image, or when a new flex instance is deployed?

**Comment by sharifelgamal** [comment]
Structure tests get run anytime the image is built and will halt the build if it fails.

**Comment by kjin** [comment]
OK, sounds good!

File: test/test_config.yaml [hunk 1]
```diff
@@ -23,3 +23,7 @@ fileExistenceTests:
   path: '/node_modules/semver'
   isDirectory: true
   shouldExist: true
+
+licenseTests:
+- debian: true
+  files: ['/nodejs/lib/node_modules/npm/LICENSE']
```
Comment (ofrobots) [inline test/test_config.yaml]: Should also be checking the Node.js license file and the semver license?

Comment (sharifelgamal) [inline test/test_config.yaml]: Yep. Added.

**Pull request merged**